analyses/20_single-cell/23_tf_pw.py

```diff
@@ -50,8 +51,55 @@
 adata_nsclc = sc.read_h5ad(path_adata_nsclc)
 
 # %%
-adatas = {"m": {}, "nsclc": {}}
+# Make pseudobulk
+pb = dc.get_pseudobulk(
+    adata_nsclc, sample_col="patient", groups_col="cell_type", mode="sum", min_cells=10, min_counts=1000
+)
+
+sc.pp.normalize_total(pb, target_sum=1e6)
+sc.pp.log1p(pb)
+
+
+# %%
+dorothea = dc.get_dorothea(organism="human")
+progeny = dc.get_progeny(organism="human", top=500)
```

Comment:
@czackl, how did you retrieve the reference data and did you use any cutoff values for `top`? If you used the defaults can we check they match between the R and Python version?

Reply:
We used the spacedeconv wrapper of decouple R which just downloads a progeny/collectri reference. As cutoff  500 was used too